_posts/en/newsletters/2023-02-08-newsletter.md

```diff
@@ -141,7 +142,21 @@ Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
 Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
 Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Bitcoin Core #25880][] p2p: Make stalling timeout adaptive during IBD FIXME:Xekyo
+- [Bitcoin Core #25880][] fixes a problem where a node could be unable
+  to acquire additional blocks during Initial Block chain Download
+  (IBD).  Previously, if the node responsible for delivering the next
```

Comment:
The 2s timeout is not a global timeout for each block, but only a "last resort" deadline if the node has gotten itself into a stalling situation. The specifics for how that can happen (1024 moving block window exhausted etc.)  are probably out the scope for this text, but one could misunderstand this such that peers only have 2s to provide a block as a general rule.

Reply:
Updating the text to @Xekyo's which provides the extra context.